### PR TITLE
Deprecated TAU Example usage

### PR DESCRIPTION
**Pull request by singa2000**
Signed-off-by: singa2000 <cookie@samsung.com>

### Change Description ###

Deprecated TAU Example usage

### Bugs Fixed ###
No

### API Changes ###
No

### CODE REVIEW

**Review by safir-srbd — APPROVED** [review]
@singa2000 Thank you for making the changes. I have looked through all the files and folders that you have made changes to, and everything seems to be in order.

@asm-foysal @mijin85cho I am approving this PR for merging.

**Comment by singa2000** [comment]
@asm-foysal @mijin85cho
Would you check it, please?
I would like the PR to be applied to tizen doc site as soon as possible.

**Comment by mijin85cho** [comment]
@asm-foysal As the author requested to apply to the docs.tizen.org ASAP, would you please sync the live branch to master branch when this PR is merged?

File: docs/application/web/api/6.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm [hunk 1]
```diff
@@ -16,14 +16,6 @@ <h1>Checkbox and Radio Button</h1>
...
-      onclick="
-         if (navigator.userAgent.toLowerCase().indexOf('chrome') == -1) {
-            alert('Chrome Browser required')
-         } else
-            window.open('https://code.tizen.org/demos?path=1.1%2Fexamples%2Fwearable%2FUIComponents%2Fcontents%2Fcontrols%2Fcheckbox.html', '_blank')"
-   >Use example</button>
-   </button>
 <br>
 <h2>Radio Button</h2>
 <br>
```
Comment (mijin85cho) [inline docs/application/web/api/6.5/ui_fw_api/Wearable_UIComponents/wearable_checkbox.htm:21]: Question. While checking https://github.com/Samsung/tizen-docs/pull/1671, I found the button in line 32 is not removed.
Same issue to 6.0
Is it OK to keep it?

**Pull request merged**